db/schema.rb

```diff
@@ -773,6 +774,16 @@
     t.datetime "updated_at", precision: nil, null: false
   end
 
+  create_table "microservice_registrations", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.string "competition_id"
+    t.integer "user_id"
+    t.text "roles"
+    t.boolean "is_competing", default: true, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["competition_id", "user_id"], name: "index_microservice_registrations_on_competition_id_and_user_id", unique: true
+  end
+
```

Comment:
Whoopsie?

Reply:
Ah this was probably because I pushed during some migration mayhem - should be resolved now?